package.json

```diff
@@ -53,7 +53,7 @@
     "lint-fix:markdownlint": "markdownlint --fix '**/*.md'",
     "lint:prettier": "prettier-standard --check",
     "lint-fix:prettier": "prettier-standard --format",
-    "test": "mocha",
+    "test": "mocha -r @babel/register test src/modules",
```

Comment:
@robertsLando I had to change this to run (existing and new) the tests in `src/modules` but I'm not super happy about this. Maybe we should talk about how want the tests to be written and where we want them to be located.
This should also take into account (future) testing of Vue components.

Reply:
@ahochsteger I would prefer to have all tests all in one folder, we could create two separete scripts to run them like `test:ui` and `test:server` and use `npm-run-all` to run them when running `npm run test`

Reply:
Yes, that makes sense, I'll move the tests from `src/modules` to `test/modules`.
So should we move the code from within `src/modules` to `libs` as well?
It's not really clear for me what goes into `lib` and what into `src/modules`.
Seem to serve the same purpose for me, or not?

Reply:
I would rename `src/modules` to `src/lb` and place there all the `js/ts` files that can be reused in the code. 

For components in general I prefer to create a folder for each one (with the name of the component) and inside that insert all component related files (.vue, .js .css) like you did for nodes table

For tests I would keep them all inside `test` folder (from root dir) and split frontend and backend tests there in 2 folders

Reply:
I guess you meant `src/lib` instead of `src/lb`, right?
Does "place there all the js/ts files that can be reused in the code" also include the files within `lib` (from root)?
If not, what's the difference between `lib` and `src/lib`?

Reply:
> I guess you meant src/lib instead of src/lb, right?

Oh yep sorry. typo :laughing: 

> Does "place there all the js/ts files that can be reused in the code" also include the files within lib (from root)?

Noipe, just the files used in frontend :) I prefer to keep front and backend splitted

> If not, what's the difference between lib and src/lib?

Think about the root folder as everything that is backend related, everything in `src` is frontend related. Also all imports oof components use `/@/` that is used to point to the `src` folder

Reply:
FYI: your first comment (using commands `test:ui` and `test:server` and `npm-run-all` for `test`) is done.
As I said, I'd do the restructuring of the tests in a separate PR.